lib/src/excel.dart

```diff
@@ -45,7 +46,9 @@ class ExcelDart {
   /// entries in the [insertionOrderList]
   static void populate(List<InsertionOrder> insertionOrderList) async {
     await Office.onReady(allowInterop((info) async {
-      await ExcelJS.run(allowInterop((context) {
+      final tableName = 'Performance_Data_Table';
```

Comment:
Table name also needs to be distinct, adding this to #36. Maybe new sheet and new table can share a same name, since they both need to be distinct?